lib/routes/account.js

```diff
@@ -492,13 +494,20 @@ module.exports = function (
           if (shouldSendNewDeviceLoginEmail) {
             // The response doesn't have to wait for this,
             // so we don't return the promise.
-            mailer.sendNewDeviceLoginNotification(
-              emailRecord.email,
-              userAgent.call({
-                acceptLanguage: request.app.acceptLanguage,
-                timestamp: Date.now()
-              }, request.headers['user-agent'])
-            )
+            getGeoData(ip, request)
```

Comment:
if `request` isn't [used by the geo-db](https://github.com/mozilla/fxa-auth-server/pull/1334/files#r72080978), might as well not pass it.